### PR TITLE
Basic UI of the school library

### PR DESCRIPTION
**Pull request by Pazzo97**
### In this PR I did the following:

- Created an app.rb file that serve as console app entry-point. It has methods that do the following:
---List all books.
---List all people.
---Create a person (teacher or student, not a plain Person).
---Create a book.
---Create a rental.
---List all rentals for a given person id.

- In main.rb define the entry point, this will be a method called main that is invoked at the end of file. This method do the following:
---Present the user with a list of options to perform.
---Lets users choose an option.
---If needed, ask for parameters for the option.
---Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,143 @@
+require_relative './person'
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+
+class App
+  def initialize
+    @persons = []
+    @books = []
```
Comment (keneogu) [inline app.rb:10]: initialize rentals too.
@rentals = []

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    print 'Enter the author of the book: '
+    author = gets.chomp.to_s
+    book = Book.new(name, author)
+    @books << book
+    puts 'Book created successfully'
+  end
+
+  # rubocop:disable Style/Next
+
+  def list_rentals_by_person_id
```
Comment (keneogu) [inline app.rb:111]: the list_rentals_by_person_id method is not working. Check your code and fix the error.

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @Pazzo97,

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] List all person for a given id is not working please fix this.
 
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ahmetbozaci — APPROVED** [review]
# 🥳 🎉🥇 💯  💥 🎆 🎈 STATUS: APPROVED 🥳 🎉  🥇  💯  💥   🎆 🎈 🎉 



@Pazzo97  ✋🏼 
#### Your project is great. 🎉 I approve of it. There is nothing else to say other than... it's time to merge it. You can submit this project on your dashboard.
___ 

### 💯 Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- I can create books, rentals, and people and ı can display them.

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
aproved

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**